src/content/docs/style-guide/writing-strategies/guidelines-documenting-ui.mdx

```diff
@@ -0,0 +1,121 @@
+
+The [APM summary page doc](/docs/apm/agents/manage-apm-agents/agent-data/triage-run-diagnostics) goes into detail about how to make use of that UI page. Note that it is focused on a specific use case, and tries to focus on how a customer would use it, instead of simply a reference of what each part does. 
+
+### Example of in-depth docs for a UI page: Distributed tracing [#example-distributed-tracing]
+
+Some UI components are harder to understand than others. This is true for the distributed tracing UI. For [that doc](/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui), we attempted to explain the most important and basic information towards the top of the doc, and then put the more detailed, technical information in collapsers at the bottom of the doc. Those “footnotes” can then be referenced in other places in the docs for when we need to give more technical detail about various aspects of that UI works. 
+
+## Examples of over-documented and redundant docs [#overdocumentation]
+
+These collapsers provide examples of excessive and unnecessary documentation:
+
+<CollapserGroup>
+  <Collapser
+    id="overdoc-example-key-transactions"
+    title="Overdocumentation example: Key transaction docs"
+  >
+In the [key transactions doc](/docs/apm/transactions/key-transactions/introduction-key-transactions) (as of August 2023) we redundantly explain many aspects of the platform that are already documented in other places. For example, we give fairly lengthy instructions on adding tags, using workloads, using Lookout, using service maps, and using deployments. All of those features are documented elsewhere and don't require inclusion in a doc about key transactions. 
+
+We also go into a lot of detail about instructions for each CRUD functionality. This could be much shorter, because once a customer knows how to create a key transaction, and where to manage it in the UI, the rest of the features will be fairly intuitive. 
+
+This doc, at this point is several pages long. Considering the simplicity of the feature (you are just making an existing transaction a key transaction), it could theoretically be just a few paragraphs. 
+</Collapser>
+  <Collapser
+    id="overdoc-example-opentelemetry"
+    title="Overdocumentation example: OpenTelemetry UI docs"
+  >
+In the OpenTelemetry UI docs (as of August 2023), we go into a lot of detail about features that are general platform experiences. 
+
+For example, in [this doc about creating custom queries](/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/view-your-data/opentelemetry-data-explorer-query-builder), we are rewriting what's already written elsewhere about these features. If we were going to write something about these features in the OpenTelemetry section, we'd want to write about OTel-specific uses of these features (for example, specific requirements, or helpful queries).
```

Comment:
I might make some of these points / tips / explanations bullet points for readability :)